File: Java/src/test/java/org/sammancoaching/UnicodeFileToHtmlTextConverterTest.java
package org.sammancoaching;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class UnicodeFileToHtmlTextConverterTest {

    @Test
    public void foo() {
        UnicodeFileToHtmlTextConverter converter = new UnicodeFileToHtmlTextConverter("foo");
        assertEquals("fixme", converter.getFilename());
    }

}

File: kotlin/src/test/kotlin/com/katas/UnicodeFileToHtmlTextConverterTest.kt
package com.katas

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class UnicodeFileToHtmlTextConverterTest {
    @Test
    fun foo() {
        val converter = UnicodeFileToHtmlTextConverter("foo")
        assertThat(converter.filename).isEqualTo("fixme")
    }
}
